careerfairsystems/arkad-app-2018
# Change map dropdown

## Change description

Replaces the dropdown on the map with an action sheet.

## How to verify

Go to the house map and press the title in the header and confirm that a action sheet is shown.

## Issues fixed

This fixes #89.
